### PR TITLE
Chore: Fixes various strict null errors

### PR DESCRIPTION
**Pull request by hugohaggmark**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: public/app/features/admin/state/reducers.test.ts [hunk 1]
```diff
@@ -70,7 +70,7 @@ describe('LDAP page reducer', () => {
                 available: true,
                 host: 'localhost',
                 port: 389,
-                error: null,
+                error: (null as unknown) as string,
```
Comment (dprokop) [inline public/app/features/admin/state/reducers.test.ts]: What is happening here? :D

**Review by dprokop — APPROVED** [review]
LGTM - should we also update ERROR_COUNT_LIMIT for strict null checks?

File: public/app/features/admin/state/reducers.test.ts (hunk 1 above)
Comment (hugohaggmark) [reply public/app/features/admin/state/reducers.test.ts]: yeah, so ugly but the recommended way according to TS :)

**Comment by hugohaggmark** [comment]
> LGTM - should we also update ERROR_COUNT_LIMIT for strict null checks?

Yes, I'll do this when this is merged so I don't break my own PR in the merge build.

File: public/app/features/admin/state/reducers.test.ts (hunk 1 above)
Comment (hugohaggmark) [reply public/app/features/admin/state/reducers.test.ts]:
```console
If this was intentional, convert the expression  to 'unknown' first.
```

Comment (dprokop) [reply public/app/features/admin/state/reducers.test.ts]: Ok, makes more sense now, thank you!

**Review by mckn — APPROVED** [review]

**Pull request merged**